### PR TITLE
Fix cards responsiveness

### PR DESCRIPTION
**Pull request by Kaleidics**
Fixes the text from spilling over into another section of the Action component. Also added an error catch to the fetch block for Action POST.

### CODE REVIEW

File: src/components/NewActionForm.js [hunk 1]
```diff
@@ -39,7 +39,11 @@ export default class NewEventForm extends Component {
                 mode: "cors",
                 headers: { "Content-Type": "application/json" },
                 body: JSON.stringify(formData)
-              }).then(this.setState({ title: "", description: "" }));
+              })
+              .then(res => res.json())
+              .then(response => console.log(response))
+              .then(this.setState({ title: "", description: "" }))
```
Comment (kelfink) [inline src/components/NewActionForm.js]: SO we're echoing out the response but not using it, right?

Comment (kelfink) [reply src/components/NewActionForm.js]: And then we empty the title and description out so the form is empty again?  I think that makes sense

**Comment by kelfink** [comment]
@Kaleidics I'll leave you to merge this then to please run through the app and make sure it all looks good from master

File: src/components/NewActionForm.js (hunk 1 above)
Comment (Kaleidics) [reply src/components/NewActionForm.js]: Yea were just checking if the server responds with 201 here

**Pull request merged**